builder/js/ncf.js

```diff
@@ -591,22 +589,20 @@ $scope.groupMethodsByCategory = function () {
   $scope.checkFilterCategory = function(methods){
```

Comment:
As the needed ng-repeat loop through an object, we can't, because Angular allow to use custom filter only on array, not on object.
> Selects a subset of items from **array** and returns it as a new array.